uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/absence-type-model.js

```diff
@@ -38,6 +40,26 @@ define([
          */
         calculateToilExpiryDate: function (absenceTypeID, date, params) {
           return absenceTypeAPI.calculateToilExpiryDate(absenceTypeID, date, params);
+        },
+
+        /**
+         * Determines if the absence type can expire by querying if
+         * the expiration unit and duration are null.
+         *
+         * @param   {string} absenceTypeId
+         * @return  {Promise}
+         */
+        canExpire: function (absenceTypeId) {
+          return absenceTypeAPI.all({
```

Comment:
Do you need to return default fields, or may be just ID since you are only checking length in the end? Do you need to return all results or just limit to 1?